### DISCUSSION TITLE
2023-07-17

### DISCUSSION THREAD

**Discussion by yeopto** [woowacourse-teams/2023-haru-study#83] [daily_meeting]
### 오늘 일정
- 근로맨들 3시~4시

### 오늘의 할 일
- 각자 할일 하기

### 내일의 할 일
- DNS 어떻게 할지

### 언젠가 해야할 일
- 기획 변경된 이유 정리

**Comment by nlom0218** [comment]
### 체크인

10점 - 한주의 시작은 즐거워요. 주말중에 깃북 블로그가 복구되어서 더 좋아요.

**Comment by yeopto** [comment]
### 체크인

9점 - 푹 쉬다와서 컨디션 이즈 굳 벗 머리가 좀 아픔

### 체크아웃

10점 - 스무스하게 일이 진행되어서 기분이 좋다.

**Comment by woosung1223** [comment]
### 체크인
9점 - 월요일이지만 스타벅스 커피도 마시고 그래서 기분이 괜찮읍니다

### 체크아웃
8점 - 컨디션은 좋으나 슬슬 눈이 피로해져서 -2점!

**Comment by jaehee329** [comment]
### 체크인
10점 - 월요일 상쾌하네요 할게 좀 많아서 무섭습니다

### 체크아웃
10점 - 월요일 상쾌하네요 할게 빨리 처리돼서 좋습니다.

**Comment by aak2075** [comment]
### 체크인

10점 - 주말 푹 쉬고와서 좋네요

### 체크아웃

9점 - 쿼리가 이렇게 나오는게 맞나요...?

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 조원들과 커피챗으로 일주일 시작하니 기분이 좋네요!

### 체크아웃
8점 - 개발에 들어가니 JPA 관련 부족한 부분이 너무 많이 보이네요... 더 열심히 해야겠다는 의미로 2점을 깎았습니다!

**Comment by woo-jk** [comment]
### 체크인
5점 - 피곤하고 배고프고 배아파서 컨디션이 안좋다🥲

### 체크아웃
9점 - 컨디션이 좋아졌다👍
